app/src/main/java/com/example/concordia_campus_guide/Adapters/PointOfInterestGvAdapter.java

```diff
@@ -0,0 +1,68 @@
+package com.example.concordia_campus_guide.Adapters;
+
+import android.content.Context;
+import android.content.res.AssetManager;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.concordia_campus_guide.R;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+class PointOfInterestGvAdapter extends BaseAdapter {
+    private Context context;
```

Comment:
Is there a way to avoid using the context ?

Reply:
I use it to link the layout the adapter will use, I do not see another way...

Reply:
ooo i see... that makes sense .